stock_production_lot_multi_company/models/stock_production_lot.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Ainara Galdona - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields
```

Comment:
Import should be alphabetically ordered ([source](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers))

`from openerp import fields, models`